local_db/api.py

```diff
@@ -0,0 +1,64 @@
+from dataclasses import dataclass
+
+from airlock.api import ProviderAPI, ReleaseRequest, User
+from local_db.models import RequestMetadata
+
+
+@dataclass
+class LocalDBProvider(ProviderAPI):
+    """Implementation of ProviderAPI using local_db models to store data."""
+
+    def _request(self, metadata: RequestMetadata = None):
```

Comment:
A matter of taste: I'd prefer to have the public interface methods at the top of the class definition.  This is so that I can look at the class and infer something about how it's going to be used.

Reply:
My default is to define them in use order, i.e. the simple ones first, and then the complex ones that use the simple ones later.

But I don't hold the that strongly at all.